CONTRIBUTING.md

```diff
@@ -0,0 +1,228 @@
+# Contributing
+
+You want to contribute and add tests to this repository? Great! We are happy to
+accept your contributions. Please read the following guidelines before you
+start.
+
+## Guidelines for contributing
+
+**TL;DR**
+
+* PR title and description should use a prefix like `[W09H03] Add behavioral tests`
+* Separate your tests logically into small files
+* Don't add tests that are not needed for the exercise
+* Write deterministic tests
+* Use JUnit 5.9.1
+* Format your code
+* Don't depend on changing the code of the implementation
+* Every test file should end with `Test.java`
+* Follow the package structure of the implementation
+* Only one test case per test method
+* Use descriptive names for test methods.
```

Comment:
```suggestion
* Use descriptive names for test methods.
* As much as possible, follow the [Java Style Guide](https://google.github.io/styleguide/javaguide.html) for variable and class notation.
```

Reply:
"Follow the Java Style Guide for variable and class notation as best as possible" sounds better imo.

Reply:
I have replaced "Format your code" with "Follow the Java Style Guide"